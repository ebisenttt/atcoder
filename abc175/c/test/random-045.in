-451416782389951 274422270597365 875712617931312
