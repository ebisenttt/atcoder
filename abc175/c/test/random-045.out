424295835541361
